modules/persistent-storage-csi-vsphere-top-aware.adoc

```diff
@@ -0,0 +1,174 @@
+. In the `clustercsidriver` object YAML file, edit the `driverConfig` section:
+* Specify the `openshift-zone` and `openshift-region` tags that you created earlier.
+* Set `driverType` to `vSphere`.
++
+[source, terminal]
+----
+~ $ oc edit clustercsidriver csi.vsphere.vmware.com -o yaml
+----
++
+Example output:
++
+[source, terminal]
+----
+apiVersion: operator.openshift.io/v1
+kind: ClusterCSIDriver
+metadata:
+ name: csi.vsphere.vmware.com
+spec:
+ logLevel: Normal
+ managementState: Managed
+ observedConfig: null
+ operatorLogLevel: Normal
+ unsupportedConfigOverrides: null
+ driverConfig: 
+  driverType: vSphere <1>
+    vSphere:
+      topologyCategories: <2>
+      - openshift-zone
+      - openshift-region
+----
+<1> Ensure that `driverType` is set to `vSphere`.
+<2> `openshift-zone` and `openshift-region` tags created earlier in vCenter.
+
+. Verify that `CSINode` object has topology keys:
++
+[source, terminal]
+----
+~ $ oc get csinode
+----
++
+Example output:
++
+[source, terminal]
+----
+NAME                        DRIVERS     AGE
+co8-4s88d-infra-2m5vd       1           27m
+co8-4s88d-master-0          1           70m
+co8-4s88d-master-1          1           70m
+co8-4s88d-master-2          1           70m
+co8-4s88d-worker-j2hmg      1           47m
+co8-4s88d-worker-mbb46      1           47m
+co8-4s88d-worker-zlk7d      1           47m
+----
++
+[source, terminal]
+----
+~ $ oc get csinode co8-4s88d-worker-j2hmg -o yaml
+----
++
+Exmaple output:
++
+[source, terminal]
+----
+...
+spec:
+    drivers:
+    - allocatable:
+        count: 59
+    name: csi-vsphere.vmware.com
+    nodeID: co8-4s88d-worker-j2hmg
+    topologyKeys: <1>
+    - topology.csi.vmware.com/openshift-zone
+    - topology.csi.vmware.com/openshift-region
+----
+<1> Topology keys from vSphere `openshift-zone` and `openshift-region` catagory tags.
++
+[NOTE]
+=====
+`CSINode` objects might take some time to receive updated topology information. After the driver is updated, `CSINode` objects should have topology keys in them.
+=====
+
+. Create a storage policy that uses the category tags:
+.. In vCenter, from the main menu, click *Policies and Profiles*.
+.. On the *Policies and Profiles* page, in the left navigation pane, click *VM Storage Policies*.
+.. Click *CREATE*.
+.. Type a name for the storage policy.
+.. For the rules, select tags that target the desired datastore. 
++ 
+The datastores are listed in the storage compatibility table.
+
+. Create a new storage class that uses the new zoned storage policy:
+.. Click *Storage* > *StorageClasses*.
+.. On the *StorageClasses* page, click *Create StorageClass*.
+.. Type a name for the new storage class in *Name*.
+.. Under *Provisioner*, select *csi.vsphere.vmware.com*. 
+.. Under *Additional parameters*, for the StoragePolicyName parameter, set *Value* to the name of the new zoned storage policy that you created earlier.
+.. Click *Create*.
++
+For example:
++
+[source, yaml]
+----
+allowVolumeExpansion: true
```

Comment:
For clarity, we usually start the storage class definition with 
kind: StorageClass

Any reason to start with something else?

Reply:
Fixed.